### DISCUSSION TITLE
Soft tissue deformation

### DISCUSSION THREAD

**Discussion by mshtx-mdacc** [idaholab/moose#26955] [Q&A General] [answered]
Howdy, 

I'm going to try and use MOOSE to model soft tissue deformation where the tissue acts similar to fluid saturated poroelastic medium. Biot's theory of biphasic consolidation is used to represent the deformation behavior of the tissue.  A snippet of the paper I'm referencing is shown here:
![image](https://github.com/idaholab/moose/assets/161739919/a072a0d2-b7b7-4300-ad90-4ab41ad3cff2)

Are the kernels already in MOOSE sufficient or will I need to create my own? I've looked through a lot of the documentation, but there's so much to digest! I thought the Tensor Mechanics module would be useful, but I am unsure. 

Cheers, 
MSH

**Comment by GiudGiud** [comment, marked as answer]
Hello

The tensor mechanics module can be an example for how to discretize this div u_vec terms. 
First identify which variables you want to solve for. I think displacements and pressure? 

then look at finding a kernel for each term in isolation which use the right variable and let you specify the coefficients you want. I dont think the TM uses shear modulus much but I m not that familiar with that code.
Same, I dont think pressure is used much in that module. It s used a lot for geo-mechanic studies so it would appear in those kernels

Guillaume

**Reply by mshtx-mdacc** [reply]
Yes, I would be solving for displacement and pressure. Thank you for the direction. 

MSH

**Comment by lynnmunday** [comment]
This page goes over the porous flow mechanics coupling in the porous_flow module:
https://mooseframework.inl.gov/modules/porous_flow/tutorial_04.html
The porous_flow module uses actions to set up kernels and materials and it is useful to use the 
https://mooseframework.inl.gov/moose/source/problems/DumpObjectsProblem.html
to find out what it is creating.
